{"name": "Проверка входа зарегистрированного пользователя с неверным логином и паролем", "status": "passed", "steps": [{"name": "Отправляем POST запрос", "status": "passed", "parameters": [{"name": "endpoint", "value": "'/api/auth/login'"}, {"name": "data", "value": "{'email': 'fTr1zgyG@mail.ru', 'password': 'Не верные данные'}"}], "start": 1735277494010, "stop": 1735277494450}, {"name": "Проверяем статус код", "status": "passed", "parameters": [{"name": "code", "value": "401"}], "start": 1735277494450, "stop": 1735277494450}], "attachments": [{"name": "stdout", "source": "76e0f754-fa36-40fc-9163-17b6a259b025-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "key", "value": "'password'"}], "start": 1735277494010, "stop": 1735277494450, "uuid": "ff33e7f0-aa84-4e3f-8257-140bfc91f7b2", "historyId": "25f37e07bc575c239e88d023d11ef17c", "testCaseId": "0d4b7aaf7852bf668446d30a4080d733", "fullName": "tests.test_login_user.TestLoginUser#test_login_user_validation_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "DESKTOP-SQI1DNR"}, {"name": "thread", "value": "235160-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}